http-servers/src/main/java/com/palantir/remoting/http/server/JsonExceptionMapper.java

```diff
@@ -59,7 +60,9 @@ public final Response toResponse(T exception) {
                 error = SerializableError.of(message, exception.getClass(),
                         Arrays.asList(stackTrace));
             } else {
-                error = SerializableError.of(message, exception.getClass());
+                String errorId = UUID.randomUUID().toString();
+                log.error("error has been logged as {}", errorId, exception);
```

Comment:
I'd still like to clean up the quality of this error text, how about:

    log.error("Suppressed Stacktrace for Error: {}", errorId, exception)